src/PowerShellEditorServices/Console/WindowsConsoleOperations.cs

```diff
@@ -41,7 +41,27 @@ public async Task<ConsoleKeyInfo> ReadKeyAsync(CancellationToken cancellationTok
                     _bufferedKey.HasValue
                         ? _bufferedKey.Value
                         : await Task.Factory.StartNew(
```

Comment:
should this be `Task.Run`?

Reply:
Maybe.  I'd like to keep it as is in this PR though.  That's from the initial implementation David wrote, we can experiment with it down the line though.